src/components/DayPickerKeyboardShortcuts.jsx

```diff
@@ -33,15 +33,91 @@ const defaultProps = {
   phrases: DayPickerKeyboardShortcutsPhrases,
 };
 
+function getKeyboardShortcuts(phrases) {
+  return [{
+    unicode: '↵',
+    label: phrases.enterKey,
+    action: phrases.selectFocusedDate,
+  },
+  {
+    unicode: '←/→',
+    label: phrases.leftArrowRightArrow,
+    action: phrases.moveFocusByOneDay,
+  },
+  {
+    unicode: '↑/↓',
+    label: phrases.upArrowDownArrow,
+    action: phrases.moveFocusByOneWeek,
+  },
+  {
+    unicode: 'PgUp/PgDn',
+    label: phrases.pageUpPageDown,
+    action: phrases.moveFocusByOneMonth,
+  },
+  {
+    unicode: 'Home/End',
+    label: phrases.homeEnd,
+    action: phrases.moveFocustoStartAndEndOfWeek,
+  },
+  {
+    unicode: 'Esc',
+    label: phrases.escape,
+    action: phrases.returnFocusToInput,
+  },
+  {
+    unicode: '?',
+    label: phrases.questionMark,
+    action: phrases.openThisPanel,
+  },
+  ];
+}
+
 class DayPickerKeyboardShortcuts extends React.Component {
   constructor(...args) {
     super(...args);
 
-    this.onClick = this.onClick.bind(this);
+    this.keyboardShortcuts = getKeyboardShortcuts(this.props.phrases);
```

Comment:
this would also need to happen in `componentWillReceiveProps`.

Reply:
Moving this was not performance motivated, I just wanted to make render() a little more concise.  Sounds like I should just move it back.

Reply:
I think it's a great idea to move it; but if it's not in the render path, then it has to be updated in both of the places where props change.

Reply:
Ok I'll make that change in a separate changeset then 👍